include/leveled.hrl

```diff
@@ -55,6 +55,7 @@
 -record(sst_options,
                         {press_method = native
                             :: leveled_sst:press_method(),
+                        press_level = 1 :: non_neg_integer(),
```

Comment:
In leveled_bookie you define COMPRESSION_LEVEL to be 1.
I think, it would be nicer to define that default here and use it:
```suggestion
                        press_level = ?COMPRESSION_LEVEL :: non_neg_integer(),
```
Otherwise you encode the default in more than one place and that might be leading to future mistakes

Reply:
I agree.  There's quite a few other things where we end up with double-definitions of defaults, but I think it makes sense to correct them all as part of this commit.  In general all configurable items should have their defaults in `leveled.hrl`